linker/Mono.Linker/Annotations.cs

```diff
@@ -56,6 +56,18 @@ public class AnnotationStore {
 		System.Xml.XmlWriter writer;
 		GZipStream zipStream;
 
+		public void Clear ()
```

Comment:
Why do you need Clear method when you can create a new instance, or is it not really clear method if it clears only half of the state?